docs/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
+gem 'github-pages', group: :jekyll_plugins
+gem "jekyll-readme-index"
```

Comment:
Good to know jekyll
https://help.github.com/en/articles/setting-up-your-github-pages-site-locally-with-jekyll